custom_components/tuya_local/devices/homebase_oscillating_fan.yaml

```diff
@@ -0,0 +1,70 @@
+name: 12" Oscillating Smart Fan
```

Comment:
As the same controller is likely to be used with other sizes of fan, I would remove the 12" from here (you can move it to the model below).